docs/user_guide/bank/getting_started_access_levels.md

```diff
@@ -35,7 +33,9 @@ Within that scope, they can:
 - Edit their Partner profile
 - Submit Requests (if the Partner has been approved)
 - View their Distributions
-- Administer Families and Children (only if they are allowed to make Child Requests (this can be limited by the Org Admin,  see [Customization](getting_started_customization.md) )) (note,  the other user types cannot see this information)
+- Administer Families and Children (only if they are allowed to make Child Requests (this can be limited by the Org Admin,  see [Customization](getting_started_customization.md) )) 
+
+- [! Note] the Banks cannot see this information
```

Comment:
Callout syntax wants a specific format. Doesn't appear to be nestable under bullet list items 🤔.

(see https://github.com/orgs/community/discussions/16925)

```suggestion
> [!NOTE]
> The Banks cannot see this information
```